workspaces/announcements/plugins/announcements/report.api.md

```diff
@@ -39,12 +39,16 @@ export const AnnouncementsCard: ({
   category,
   active,
   variant,
+  sortBy,
+  order,
 }: {
   title?: string | undefined;
   max?: number | undefined;
   category?: string | undefined;
   active?: boolean | undefined;
   variant?: InfoCardVariants | undefined;
+  sortBy?: 'created_at' | 'start_at' | undefined;
+  order?: 'desc' | 'asc' | undefined;
```

Comment:
non-blocking: 95% sure all of the `| undefined`s on this type are unnecessary. Making the property optional handles that. Up to you to change it for your contributions, but not needed for the existing props (unless you want to create a separate pr 😄 )